lib/delve.js

```diff
@@ -26,7 +26,19 @@ function addOutputMessage(messageType, message) {
 }
 
 function run(file, method) {
-	const { path: dlvPath, args } = store.getState().delve;
+	const { path: dlvPath, args, fileOverride } = store.getState().delve;
+
+	let trimmedFile = fileOverride.trim();
+	if (fileOverride && trimmedFile.trim !== "") {
+		const editor = atom.workspace.getActiveTextEditor();
+		const relativizedPath = atom.project.relativizePath(editor.getPath());
+		if (relativizedPath.length === 2) {
```

Comment:
I don't get this... what'st he point of this _relativize_ and _resolve_ steps? can you give me some examples where this if is either true or false and how this could be of any use?

Isn't it enough to something like:
```js
if (path.isAbsolute(trimmedFile)) {
    file = trimmedFile
} else {
    file = atom.project.resolvePath(trimmedFile);
}
```